Dictionaries/eng_OCRFixReplaceList.xml

```diff
@@ -852,22 +840,22 @@
     <Word from="Lsn't" to="Isn't" />
     <Word from="Lst's" to="Let's" />
     <Word from="lsuppose" to="I suppose" />
-    <Word from="lt" to="it" />
+    <Word from="lt" to="It" />

```

Comment:
But that's the usual `I`/`l` OCR error. It's much more likely those two will be confused, because they look exactly the same, then `i` and `I`.